src/scss/includes/search_form.scss

```diff
@@ -233,11 +233,13 @@ input[type="search"] {
 
     &.autocomplete_suggestion__first_line {
       color: $primary_text;
+      line-height: 21px;
     }
 
     &.autocomplete_suggestion__second_line {
       color: $secondary_text;
       font-size: 14px;
+      line-height: 18px;
```

Comment:
Not a big deal, but has this been approved by the design team yet?
In the meantime, we can remove this change from this PR if we want to merge.